Add figure captions (#561)
worldbank/dime-data-handbook: chapters/4-acquisition.tex

Finally, it emphasized the secure handling of this potentially sensitive information,
giving you a range of tools and a complete workflow
for transferring and storing data securely at all times.
Figure 2 shows an overview of the tasks and outputs discussed. 

> Note that the data acquisition process often occurs repeated times, 
> and commonly takes both of the paths portrayed in the figure in different occasions, 
> particularly when multiple data sources are involved.

Reviewer: ```suggestion
Note that projects using multiple data sources will 
repeat the process portrayed in the figure multiple times, 
and may take both of the paths portrayed in the figure on different occasions. 
```
